documentation/src/docs/admin-guide/cluster-dependencies.md

```diff
@@ -13,7 +17,6 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-# Cluster Service Dependencies
```

Comment:
Aren't these also anchors in the generated HTML document? If we have links pointing to them, they won't jump to the right section of the page anymore.

Reply:
Yes, in other docs, I've found that we don't have to change `#` titles to `title` front-matter tables. These particular changes come from Claudio, and maybe we can revert them. It does change how the page renders in github source view also... In either case, however, Docusaurus doesn't provide an anchor on the main title, whether using a `#` title or a front-matter table `title`.

Compare https://pravega.io/docs/snapshot/pravega-operator/manual-installation
Vs https://pravega.io/docs/snapshot/pravega/admin-guide/cluster-dependencies

Reply:
`#`-based main titles have been restored